data_steward/utils/bq.py

```diff
@@ -15,11 +17,41 @@ def get_client(project_id=None):
         logging.info(f'You should specify project_id for a reliable experience.'
                      f'Defaulting to {os.environ.get(GOOGLE_CLOUD_PROJECT)}.')
         return bigquery.Client()
-    else:
-        return bigquery.Client(project=project_id)
+    return bigquery.Client(project=project_id)
 
 
 def query(q, project_id=None, use_cache=False):
     client = get_client(project_id)
     query_job_config = bigquery.job.QueryJobConfig(use_query_cache=use_cache)
     return client.query(q, job_config=query_job_config).to_dataframe()
+
+
+def list_datasets(project_id):
+    """
+    Lists all datasets existing in a project.
+
+    :return: List of dataset objects
+    """
+    client = get_client(project_id)
+    datasets = list(client.list_datasets())
+    return datasets
+
+
+def delete_dataset(project_id,
+                   dataset_id,
+                   delete_contents=True,
+                   not_found_ok=True):
+    """
+    Delete a dataset in a project
+
+    :param project_id: Identifies the project the containing the dataset
+    :param dataset_id: Identifies the dataset to delete
+    :param delete_contents: If set True, deletes all contents within the dataset
+    :param not_found_ok: If set True, does not raise error if dataset cannot be found
```

Comment:
I know we can easily check the signature, but I would specifically comment about the default value for these parameters.  This will be important if/when we start using documentation tools.

Reply:
Yes, that is a good point